lib/events.js

```diff
@@ -229,8 +229,25 @@ class AdobeIOEvents {
      * @param {RetryOptions} retryOptions retry options. If set to false, retry functionality will be disabled. If not set, it will use the default times
      * @returns {Promise}
      */
-    async sendEvent(event, retryOptions=true) {
+    async sendEvent(event, retryOptions) {
         const url = `${INGRESS_HOST[this.env]}/api/events`;
+        function retryOnHttpResponse(response) {
+            // retry on any 5xx status codes and on 204(IO events cache issue)
+            if (response && (response.status === 204 || response.status >= 500 )) {
+                return true;
+            }
+            return false;
+        }
+        if (typeof retryOptions === 'object') {
+            if(!retryOptions.retryOnHttpResponse) {
+                retryOptions.retryOnHttpResponse = retryOnHttpResponse;
+            }
+        } else {
+            retryOptions = {
```

Comment:
Just want to verify that we want to force a retry here, even if `retryOptions` is set to `false`. Not sure if anyone is using the method this way, but the jsdoc comments make it sound possible. If we are going to force a retry, we may want to update the jsdoc comments.

Reply:
yea, have been thinking about it too. This method is not called with retry `false` anywhere. Will review this part to see if `false` can still be supported.